aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -56,7 +56,36 @@ Tensor sum_batching_rule(const Tensor& self, IntArrayRef dims, bool keepdim, opt
 }
 
 Tensor mul_batching_rule(const Tensor& self, const Tensor& other) {
-  auto physical_args = BroadcastingVmapTransform::logicalToPhysical({self, other});
```

Comment:
Don't you have to apply this change to everything, not just mul?

Reply:
mul is actually the only binary broadcasting pointwise operator that has a batching rule right now. I'm going to add the other ones in a later PR (and they'll have this fix).